src/index-metadbs/jarvis/v1/info.json

```diff
@@ -0,0 +1,42 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
```

Comment:
Shouldn't we update this for all now?
This is not a change that should be implemented in this PR though, but wanted to say it while I was here.

Reply:
Yeah, thought about this too. Could you raise an issue?